src/helm/benchmark/metrics/omni_math_metrics.py

```diff
@@ -19,7 +19,11 @@ def evaluate_generation(
         eval_cache_path: str,
     ) -> List[Stat]:
         assert request_state.annotations
-        score = request_state.annotations["omni_math"]["equivalence_judgement"].strip().upper() == "TRUE"
+        all_judgements = request_state.annotations["omni_math"]["equivalence_judgement"]
+        if len(all_judgements) == 0:
+            raise ValueError("Could not compute Omni-MATH accuracy because all annotators failed.")
+        judgement_bools = [judgement.strip().upper() == "TRUE" for judgement in all_judgements]
+        score = sum(judgement_bools) / len(judgement_bools)
```

Comment:
It's not valid to sum an array of bools, right? You need to cast them to int first.

Reply:
I think bool is a subclass of int in Python, so it actually works fine..? If that introduces too much ambiguity I can go with explicit casting for sure.

```
>>> sum([True, False])
1
>>> issubclass(bool, int)
True
```